src/displayapp/screens/WatchFaceAnalog.cpp

```diff
@@ -60,9 +58,44 @@ WatchFaceAnalog::WatchFaceAnalog(Controllers::DateTime& dateTimeController,
   sMinute = 99;
   sSecond = 99;
 
-  lv_obj_t* bg_clock_img = lv_img_create(lv_scr_act(), nullptr);
-  lv_img_set_src(bg_clock_img, &bg_clock);
-  lv_obj_align(bg_clock_img, nullptr, LV_ALIGN_CENTER, 0, 0);
+  minor_scales = lv_linemeter_create(lv_scr_act(), nullptr);
+  // lv_linemeter_set_range(minor_scales, 1, 60);
```

Comment:
Oh, I have to remove them. Initially I thought this would be the value range for minutes, but since the widget is purely decorative we don't need to set this.